mysql/datadog_checks/mysql/config_models/defaults.py

```diff
@@ -96,6 +96,14 @@ def instance_ssl(field, value):
     return get_default_field_value(field, value)
 
 
+def instance_additional_status(field, value):
```

Comment:
This file is autogenerated. Changes here need to come from the [spec file ](https://github.com/DataDog/integrations-core/blob/master/mysql/assets/configuration/spec.yaml) and then synced with `ddev validate models -s`